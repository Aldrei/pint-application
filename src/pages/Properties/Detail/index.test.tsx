import React from 'react';

import { render } from '@testing-library/react';

import PropertyDetail from './index';

jest.mock('../../../hooks/useReducerSelector', () => ({
  useAppSelectorBlaBlaBal: jest.fn(() => ({}))
}));

jest.mock('../../../stores/hooks', () => ({
  useAppDispatch: jest.fn
}));

describe('Property detail page', () => {
  it('Should render correctly', () => {
    const nodeEl = render(<PropertyDetail />);
    expect(nodeEl.baseElement).toMatchSnapshot();
  });
});
