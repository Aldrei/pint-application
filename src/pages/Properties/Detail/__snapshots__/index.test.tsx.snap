// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Property detail page Should render correctly 1`] = `
<body>
  <div>
    <div
      class="MuiBox-root css-1djy331"
      data-testid="propertiesList-container"
    >
      <ul
        class="MuiImageList-root MuiImageList-standard css-rvv586-MuiImageList-root"
        style="grid-template-columns: repeat(1, 1fr); gap: 4px;"
      />
      <ul
        class="MuiList-root MuiList-padding css-h4y409-MuiList-root"
        style="width: 100%;"
      />
    </div>
  </div>
</body>
`;
